lib/open_food_network/products_renderer.rb

```diff
@@ -4,19 +4,21 @@ module OpenFoodNetwork
   class ProductsRenderer
```

Comment:
it would be great to see this file flying to app/services

Reply:
I was just thinking that!